src/emm/emm_cli.py

```diff
@@ -211,5 +220,15 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_context_width=100))
+@click.argument("commit", nargs=-1)
+@click.option("--directory", default=None, help="Directory to execute command at.")
+@click.pass_context
+def git_commit(ctx: click.Context, commit: str, directory: str) -> None:
+    """Perform git commit -a operation to all modules."""
+    orchestrator = EmmOrchestrator()
+    orchestrator.git_commit_modules(commit, directory)
```

Comment:
Instead of `commit`, can we use `commit-message` or something like that? Just `commit` is ambiguous and could refer to a number of things. (This commit applies to all the uses of `commit` there were added in this PR.